### ISSUE TITLE
Register project at packagist

### ISSUE THREAD

**Issue by tored** [c9s/PHPRelease#11]
Simplifies use of project in a strict composer work flow.

**Comment by c9s** [comment]
Is there an API for this?

**Comment by svenax** [comment]
Yes, register an account at packagist.org and activate the "packagist" service hook for your Github repo. Simple as that.
